lunamark/util.lua

```diff
@@ -224,5 +224,62 @@ function M.escaper(char_escapes, string_escapes)
   end
 end
 
+--- Given a Roman number in uppercase, return its decimal value.
+function M.roman2number(roman)
```

Comment:
I suggest changing `2` to `_to_` for consistency with existing functions, see e.g. `util.rope_to_string()`.

Reply:
```suggestion
function M.roman_to_number(roman)
```

Reply:
Done (here and when used)